test/test_fake_tensor.py

```diff
@@ -205,6 +205,14 @@ def test_mode(self):
 
         self.assertTrue(isinstance(out, FakeTensor))
 
+    def test_full(self):
+        # Test torch.full returns tensor with correct dtype
+        with FakeTensorMode():
```

Comment:
nit: just run with `with torch._subclasses.CrossRefFakeMode():`, and that will do testing for you (and more exhaustively).